client/i18n/locales/english/translations.json

```diff
@@ -376,6 +376,7 @@
     "tests-completed": "// tests completed",
     "console-output": "// console output",
     "syntax-error": "Your code raised an error before any tests could run. Please fix it and try again.",
+    "indentation-error": "Your code has an indentation error. You may need to add 'pass' on a new line to form a valid block of code.",
```

Comment:
```suggestion
    "indentation-error": "Your code has an indentation error, meaning the line spacing isn't consistent. Ensure all lines in a code block are indented the same way.",
```

Reply:
The message is thought mainly for the case in which camper need to add `pass`.

Reply:
This seems like a very specific use case to me? Or am I wrong @ojeytonwilliams ?

Reply:
It is, yes. The problem is that some python lessons will have instructions like "create a function `foo`", but if learners just write

```py
def foo():
```

that's not valid python. In this case the hint will appear and, hopefully, they will write

```py
def foo():
  pass
```

which is valid python.

Essentially it's an alternative strategy to writing "please remember to include `pass` in your function stub", or similar, in every lesson like this. We may find that it's not an effective strategy, but I think it's worth trying for now.